CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # History
 
+#### 5.0.0 Upgrade Notes
+
+- Deprecating support for extensionless files. (**Intervention needed when updating from < 5.0.0!**)
```

Comment:
Extensionless files are still supported; they are just not assumed to be Turtle anymore.

How about "As of v5.0.0, all Turtle files need an extension"